src/main/java/com/splunk/hecclient/HecAckPoller.java

```diff
@@ -132,7 +132,7 @@ public void add(HecChannel channel, EventBatch batch, String response) {
         }
 
         if (channelEvents.get(resp.getAckId()) != null) {
-            log.error("ackId={} already exists for channel={} index={}", resp.getAckId(), channel, channel.getIndexer());
+            log.warn("ackId={} already exists for channel={} index={}, possible duplication of data", resp.getAckId(), channel, channel.getIndexer());
```

Comment:
Modify to             
`log.warn("ackId={} already exists for channel={} index={} data may be duplicated in Splunk", resp.getAckId(), channel, channel.getIndexer());`